server/src/test/java/io/crate/integrationtests/GroupByAggregateTest.java

```diff
@@ -1393,4 +1393,25 @@ public void test_group_by_on_duplicate_keys() throws Exception {
             "122021430M| Rue Penavayre| 122021430M| Pt(x=2.573608970269561,y=44.35006999410689)| Rodez\n"
         ));
     }
+
+    @Test
+    public void test_group_by_of_constant_null() {
+        execute("create table tbl (x int)");
```

Comment:
not sure about those tests (should be dropped?) - I was confused by planner results (https://github.com/crate/crate/pull/12582/files#r888624169) and verified results by `test_group_by_of_constant_aliased_null`